runs/comparisons/rf.py

```diff
@@ -29,7 +34,7 @@
         X_train=X_train,
         y_train=y_train,
         random_state=random_state,
-        n_calls=10,
+        n_calls=1000,
```

Comment:
Might be the phone but why is there an indent here?

Reply:
I can't see an indent, so it's probably the phone